<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qtilities::MainDialog</class>
 <widget class="QDialog" name="Qtilities::MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="layControls">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbxBright">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>35</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblBright">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblTime">
       <property name="text">
        <string>Time range</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sbxTemp">
       <property name="suffix">
        <string notr="true">K</string>
       </property>
       <property name="minimum">
        <number>3200</number>
       </property>
       <property name="maximum">
        <number>6500</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblBacklight">
       <property name="text">
        <string>Backlight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbxBacklight">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>35</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblTemp">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QTimeEdit" name="timeEnd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="time">
        <time>
         <hour>16</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTimeEdit" name="timeStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="time">
        <time>
         <hour>6</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="layBtnStartStop">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="vspColors">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbnAbout">
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset theme="help-about"/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnStartStop">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start"/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbxBacklight</tabstop>
  <tabstop>sbxBright</tabstop>
  <tabstop>sbxTemp</tabstop>
  <tabstop>timeStart</tabstop>
  <tabstop>timeEnd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
